src/components/blocks/software/Pre.test.tsx

```diff
@@ -42,4 +70,50 @@ describe('<Pre />', () => {
     const { container } = render(<Pre data={PRE_TEST_DATA} language="javascript" languages={undefined} />);
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  it('should successfully render code elements with both Rest and Realtime languages', () => {
+    const { container } = render(
+      <Pre
+        data={PRE_REALTIME_JS_TEST_DATA}
+        language="javascript"
+        languages={['realtime_javascript', 'realtime_csharp', 'rest_javascript', 'rest_csharp']}
+        altData={{
+          realtime_javascript: PRE_REALTIME_JS_TEST_DATA,
+          realtime_csharp: PRE_REALTIME_CSHARP_TEST_DATA,
+          rest_javascript: PRE_REST_JS_TEST_DATA,
+          rest_csharp: PRE_REST_CSHARP_TEST_DATA,
+        }}
+        isSDKInterface={true}
+        realtimeAltData={{
+          realtime_javascript: PRE_REALTIME_JS_TEST_DATA,
+          realtime_csharp: PRE_REALTIME_CSHARP_TEST_DATA,
+        }}
+        restAltData={{
+          rest_javascript: PRE_REST_JS_TEST_DATA,
+          rest_csharp: PRE_REST_CSHARP_TEST_DATA,
+        }}
+      />,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  it('should successfully render code elements with  only Realtime languages', () => {
```

Comment:
Could we add a scenario here for Rest only as well, please?

Reply:
@artismarti  pushed and added